phys2bids/tests/test_integration.py

```diff
@@ -0,0 +1,466 @@
+                      log_info) if 'Sampling Frequency' in log_str]
+    sampling_found = log_info[sampling_idx[0]]
+    assert '1000.0' in sampling_found
+
+    # Check sampling frequency
+    started_idx = [log_idx for log_idx, log_str in enumerate(
+                      log_info) if 'Sampling started' in log_str]
+    started_found = log_info[started_idx[0]]
+    assert '0.24499999999989086' in started_found
+
+    # Check start time
+    start_idx = [log_idx for log_idx, log_str in enumerate(
+                      log_info) if 'first trigger' in log_str]
+    start_found = log_info[start_idx[0]]
+    assert 'Time 0' in start_found
+
+    # Checks json file
+    json_filename = 'sub-006_ses-01_task-test_rec-labchart_run-00_physio.json'
+    with open(os.path.join(test_path_output, json_filename)) as json_file:
+        json_data = json.load(json_file)
+
+    # Compares values in json file with ground truth
+    assert json_data['SamplingFrequency'] == 1000.0
+    assert json_data['StartTime'] == 0.24499999999989086
+    assert json_data['Columns'] == ['time', 'Trigger', 'CO2', 'O2', 'Pulse']
+
+    # Remove generated files
+    for filename in glob.glob(os.path.join(test_path, 'phys2bids*')):
+        os.remove(filename)
+    for filename in glob.glob(os.path.join(test_path, 'Test_belt_pulse_samefreq*')):
+        os.remove(filename)
+    for filename in glob.glob(os.path.join(test_path_output, '*')):
+        os.remove(filename)
+
+
+def test_integration_info():
+    """
+    Tests the info option
+    """
+    test_path = resource_filename('phys2bids', 'tests/data')
+    test_filename = 'tutorial_file.txt'
+    test_chtrig = 1
+    test_outdir = test_path
+    test_ntp = 158
+    test_tr = 1.2
+    test_thr = 0.735
+
+    # Move into folder
+    subprocess.run(f'cd {test_path}', shell=True, check=True)
+
+    # Phys2bids call through terminal
+    command_str = (f'phys2bids -in {test_filename} -indir {test_path} ',
+                   f'-chtrig {test_chtrig} -outdir {test_outdir} ',
+                   f'-tr {test_tr} -ntp {test_ntp} -thr {test_thr} ',
+                   f'-info')
+    command_str = ''.join(command_str)
+
+    subprocess.run(command_str, shell=True, check=True)
+
+    # Check that plot all file is generated
+    assert os.path.isfile('tutorial_file.png')
+
+    # Read logger file
+    files = os.listdir(test_path)
+    logger_idx = [log_idx for log_idx, log_str in enumerate(
+                      files) if 'phys2bids' in log_str]
+    logger_file = files[logger_idx[0]]
+
+    # Check files were correctly read
+    with open(os.path.join(test_path, logger_file)) as logger_info:
+        logger_info = logger_info.readlines()
+
+    # Get trigger info
+    trigger_idx = [log_idx for log_idx, log_str in enumerate(
+                      logger_info) if 'Trigger; sampled at' in log_str]
+    trigger_found = logger_info[trigger_idx[0]]
+    assert '1000.0' in trigger_found
```

Comment:
Either the for-loop or helper function for checking these outputs would be ideal!